src/main/resources/config/application-test.properties

```diff
@@ -22,4 +22,10 @@ spring.thymeleaf.view-names= *
 spring.thymeleaf.suffix=.html
 spring.thymeleaf.mode=HTML5
 spring.thymeleaf.encoding=UTF-8
-spring.thymeleaf.content-type=text/html
\ No newline at end of file
+spring.thymeleaf.content-type=text/html
+
+##AREA FIFTYLAN SETTINGS
+
+a5l.paymentReturnUrl=https://areafiftylan.nl/ordersuccess
```

Comment:
This will break if the server is run on a different path, e.g. `v1/api/`. Would it be possible to base the url on the root path or must it be manually updated?